src/en/reference-install.md

```diff
@@ -161,15 +161,15 @@ instructions on how to build Juju from source.
 
 <!-- LINKS -->
 
-[release-notes-2]: ./reference-release-notes.html
+[release-notes-2]: ./reference-release-notes.md
 [homebrew]: https://brew.sh/
 [contributing]: https://github.com/juju/juju/blob/develop/CONTRIBUTING.md
 [snapcraft]: https://snapcraft.io
 [snapd-install]: https://snapcraft.io/docs/core/install
 [juju-new-bug]: https://bugs.launchpad.net/juju/+filebug
-[juju-win-signed]: https://launchpad.net/juju/2.3/2.3.8/+download/juju-setup-2.3.8-signed.exe
-[juju-win-signed-md5]: https://launchpad.net/juju/2.3/2.3.8/+download/juju-setup-2.3.8-signed.exe/+md5
-[juju-centos]: https://launchpad.net/juju/2.3/2.3.8/+download/juju-2.3.8-centos7.tar.gz
-[juju-centos-md5]: https://launchpad.net/juju/2.3/2.3.8/+download/juju-2.3.8-centos7.tar.gz/+md5
+[juju-win-signed]: https://launchpad.net/juju/2.3/2.4.0/+download/juju-setup-2.4.0-signed.exe
```

Comment:
all of these look wrong - why is there a 2.3 in there?

Reply:
Oops.